File: Exemplos_desafios_linguagens/calculoEmJavascrip/calculo.js
const valorSalario = 2000;
const valorBeneficios = 250;

const valorImposto = () => {
    
    let aliquota = 0;
    if(valorSalario > 0 && valorSalario <= 1100) {
        aliquota = 0.05;
    }else if (valorSalario > 1100 && valorSalario <= 2500) {
        aliquota = 0.1;
    }else {
        aliquota = 0.15;
    }
    return aliquota * valorSalario;
}

const saida = valorSalario - valorImposto() + valorBeneficios;

console.log(`Seu salário liquido é de R$ ${saida.toFixed(2)}`);

File: Exemplos_desafios_linguagens/desafioC#/calculo.cs
using System;

namespace calculo
{
    class Program
    {
        static void Main(string[] args)
        {
            console.WriteLine("Digite o valor do seu salário: ");
            float valorSalario = float.Parse(Console.ReadLine());
            console.WriteLine("Digite o valor dos benefícios: ");
            float valorBeneficios = float.Parse(Console.ReadLine());

            float valorImposto = 0;

            if (valorSalario >= 0 && valorSalario <= 1100)
            {
                valorImposto = 0.05f * valorSalario;
            }
            else if (valorSalario >= 1100.01f && valorSalario <= 2500)
            {
                valorImposto = 0.1f * valorSalario;
            }
            else
            {
                valorImposto = 0.15f * valorSalario;
            }
            float saida = valorSalario - valorImposto + valorBeneficios;
            console.WriteLine("Salário líquido: " + saida.ToString("F2"));
        }
    }
}